### PR TITLE
Separate variable for ModuleName

### PR DESCRIPTION
**Pull request by Glober777**
Moved out the module name into a separate variable (PowerShell/Plaster/#243)

### CODE REVIEW

File: src/Templates/NewPowerShellScriptModule/test/Module.T.ps1 [hunk 1]
```diff
@@ -1,4 +1,5 @@
-$ModuleManifestName = '<%=$PLASTER_PARAM_ModuleName%>.psd1'
+$ModuleName = '<%=$PLASTER_PARAM_ModuleName%>'
+$ModuleManifestName = "$ModuleName.psd1"
```
Comment (rkeithhill) [inline src/Templates/NewPowerShellScriptModule/test/Module.T.ps1]: Can you change the value to `"${ModuleName}.psd1"?  It reads a bit better.  Thanks.

Comment (Glober777) [reply src/Templates/NewPowerShellScriptModule/test/Module.T.ps1]: I've been thinking about that too initially, but ScriptAnalyzer thinks that the ```$ModuleName```posh variable is not in use and tags it accordingly. So, should we still go with ```${ModuleName}```posh instead?

Comment (Glober777) [reply src/Templates/NewPowerShellScriptModule/test/Module.T.ps1]: @rkeithhill, I'm afraid that I'm going to need your answer before proceeding :)

Comment (rkeithhill) [reply src/Templates/NewPowerShellScriptModule/test/Module.T.ps1]: If my suggested change triggers a PSSA warning then no, don't change it.

**Pull request merged**